<launch>
  <node name="controller" pkg="follow_path" type="PID_controller.py" output="screen">
    <param name="kp_ang" value="30"/>
    <param name="ki_ang" value="0"/>
    <param name="kd_ang" value="15"/>
    <param name="kp_lin" value="0.6"/>
    <param name="ki_lin" value="0.01"/>
    <param name="kd_lin" value="6"/>
    <param name="path_type" value="0"/>
  </node>
  
  <include file="$(find follow_path)/launch/simulation_manager.launch"/>
</launch>